docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -23,30 +29,104 @@ so the database connections are correctly configured.
+party-server you do not need to set anything. If you have deployed NATS yourself or are running multiple Party Servers,
+you will need to set the environment variable:
 
 - `nats.urls` - a comma separated list of NATs servers.
 
-NATS works by having the clients tell the servers where each other
-are, so the NATS servers need to be routable (i.e. they must be able to talk to each other) but do not need to
-be explicitly told about each other.
+You do not need to configure anything further for NATS for Dacha1 or Dacha2. NATS supports request/reply, 
+true pub/sub and pub/queues in a single product. 
+
+=== Google's Pub/Sub
+
+For Google's PubSub, you will need to turn nats *off*. We consider Google Pub/Sub to be production ready as of 1.5.9.
+
+- `nats.enabled=false`
+
+And PubSub on:
+
+- `cloudevents.pubsub.enabled=true`
+
+You will also need to tell FeatureHub what the topics are for publishing on. There is no harm in configuring these
+across all three applications:
+
+- `cloudevents.pubsub.project=featurehub` - whatever the Google project ID you are deploying FeatureHub into
+- `cloudevents.edge-mr.pubsub.topic-name=featurehub-edge-updates` - the name of the topic used by Edge to publish TestSDK updates back to MR
+- `cloudevents.mr-edge.pubsub.topic-name=featurehub-mr-edge` - Feature Updates published from MR -> Edge for streaming clients
+- `cloudevents.stats.pubsub.topic-name=featurehub-stats` - You only need to configure this if you have Stats publishing turned on, otherwise you can ignore it
+- `cloudevents.mr-dacha2.pubsub.topic-name=featurehub-mr-dacha2` - the name of the topic MR publishes feature updates, environment updates and service account updates.
+- `cloudevents.pubsub.min-backoff-delay-seconds=5` - Edge and Dacha must be able to create their own subscriptions. Google PubSub is not actually "pubsub", there is no way for all connections to a subscription
+to receive a message, so this delay reflects how long PubSub should wait before trying to redeliver the message. Edge and Dacha will delete their subscriptions when they shut down, so keep this low.
+- `cloudevents.mr-edge.pubsub.subscription-prefix=featurehub-edge-listener` - the prefix used for subscriptions created by Edge 
+- `cloudevents.mr-dacha2.pubsub.subscription-prefix=featurehub-dacha2-listener` - the prefix used for subscriptions created by Dacha2 instances
+
+MR also needs an extra environment variable to tell it what subscription to listen to for updates from the TestSDK topic configured above (`cloudevents.edge-mr.pubsub.topic-name`):
+
+- `cloudevents.inbound.channel-names=featurehub-edge-updates-mr-sub`
+
+For testing locally there are extra fields if you are using the emulator - they are used in the `featurehub-installs` PubSub folder. To test in Google, we use https://www.pulumi.com/[Pulumi]. 
+
+=== AWS Kinesis
+
+This is the support for AWS Kinesis. We are not fully happy with the performance of this, the polling only mode and the delivery jitter 
```

Comment:
Shall we start from saying "We are now supporting AWS Kinesis however we are not fully happy with the performance... "